hw/ip/otbn/doc/_index.md

```diff
@@ -439,6 +439,25 @@ outer_body:
 
 ## Design Details
 
+### Memories
+
+The OTBN processor core has access to two dedicated memories:
+an instruction memory (IMEM), and a data memory (DMEM).
+Each memory is 4 kiB in size.
+
+The memory layout follows the Harvard architecture.
+Both memories are byte-addressed, with addresses starting at 0.
+
+The instruction memory (IMEM) is 32b wide and provides the instruction stream to the OTBN processor;
+it cannot be read or written from user code through load or store instructions.
+
+The data memory (DMEM) is 256b wide and read-write accessible from the base and big number instruction subsets of the OTBN processor core.
+When accessed from the base instruction subset through the `LW` or `SW` instructions, accesses must read or write 32b-aligned 32b words.
+When accessed from the big number instruction subset through the `BN.LID` or `BN.SID` instructions, accesses must read or write 256b-aligned 256b words.
+
+Both memories can be accessed through OTBN's register interface ({{< regref "DMEM" >}} and {{< regref "IMEM" >}}) only when OTBN is idle, as indicated by the {{< regref "STATUS.busy">}} flag.
```

Comment:
I think the "only" in this sentence needs to move, to make "Both memories can only be accessed ... when OTBN is idle". The "as indicated" bit is a bit ambiguous too: we're trying to say that OTBN's idleness is indicated by the flag, but it could be read as saying that this part of the spec should be inferred from the documentation of that flag or something.

How about something like this?
```
Both memories can be accessed through OTBN's register interface ({{< regref "DMEM" >}} and {{< regref "IMEM" >}}).
These accesses will be ignored if OTBN is busy.
A host processor can check whether OTBN is busy by reading the {{< regref "STATUS.busy">}} flag.
```

I know that the middle sentence here isn't detailed enough, but I don't think we've agreed on the exact spec yet. Also, I realise that this PR was just moving this sentence, but it's probably still a good opportunity to improve it!